packages/components/src/internal/components/entities/utils.ts

```diff
@@ -125,3 +127,17 @@ export function getUpdatedRowForParentChanges(
     });
     return updatedValues;
 }
+
+export function getUniqueIdColumnMetadata(queryInfo: QueryInfo) : Map<string, EditableColumnMetadata> {
+    let columnMetadata = Map<string, EditableColumnMetadata>();
+    queryInfo?.columns.filter((column) => column.conceptURI === STORAGE_UNIQUE_ID_CONCEPT_URI).forEach(
```

Comment:
can the QueryColumn `isUniqueIdColumn` helper be used here? I haven't been keeping close track of the places in this PR which use `conceptURI === STORAGE_UNIQUE_ID_CONCEPT_URI` but I assume most of them could use one of the helper functions / methods you've added to various object types.

Reply:
yup